packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderItem/ConditionBuilderItemOption/ItemOptionForValueField.js

```diff
@@ -16,16 +16,17 @@ import {
   translateWithId,
 } from '../../ConditionBuilderContext/DataConfigs';
 
-export const ConditionBuilderItemOption = ({
+export const ItemOptionForValueField = ({
```

Comment:
similar issues here to address that i mentioned in `ItemOption` regarding filtering values

Reply:
modified